### DISCUSSION TITLE
OVOSSkill API documentation

### DISCUSSION THREAD

**Discussion by wita-sec** [OpenVoiceOS/ovos-buildroot#216] [Q&A] [answered]
Dear all,
where can I find documentation of the OVOS Skill API? Also example code would be great. 

Greets,
Michael

**Comment by JarbasAl** [comment, marked as answer]
I have been working in our developer documentation recently, this should be a good start https://openvoiceos.github.io/ovos-technical-manual/

**Comment by wita-sec** [comment]
Looks good, thank you.
